### PR TITLE
FAB-17410 Updated doc links to latest levels for Fabric 2.0

### PR DESCRIPTION
**Pull request by pamandrejko**
- Did not update references to 1.4 cc lifecycle
- Removed Kafka (which had old links) from FAQ

Signed-off-by: pama-ibm <pama@ibm.com>

#### Type of change
- Documentation update

#### Description

I left in references to v1.4 for lifecycle where we explicitly call it out.
I did not update SDK 1.4 references since they are not available yet. When Fabric 2.0 SDK levels are  available, we should revisit the docs and perform a similar update.

#### Related issues

FAB-17410
FAB-17411

### CODE REVIEW

File: docs/source/Fabric-FAQ.rst [hunk 1]
```diff
@@ -242,14 +242,12 @@ Ordering Service
...
-  against these interfaces already: raft_ and kafka_. You can study them to take
-  cues for your own implementation. The ordering service code can be found under
+  interfaces defined in the `consensus package`_. There is a plugin built
+  against raft_ . You can study it to learn more for your own implementation. The ordering service code can be found under
   the `orderer package`_.
 
 .. _consensus package: https://github.com/hyperledger/fabric/blob/master/orderer/consensus/consensus.go
 .. _raft: https://github.com/hyperledger/fabric/tree/master/orderer/consensus/etcdraft
-.. _kafka: https://github.com/hyperledger/fabric/tree/master/orderer/consensus/kafka
 .. _orderer package: https://github.com/hyperledger/fabric/tree/master/orderer
```
Comment (sykesm) [inline docs/source/Fabric-FAQ.rst]: Shouldn't these be pointing to release-2.0.0? Master can change at any time so I'd expect references in doc to point to the more stable release branches.

File: docs/source/cc_service.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ The chaincode package should be used to provide two pieces of information to the
 * provide chaincode endpoint information in a `connection.json` file placed in the release directory. The `bin/run` section describes the `connection.json` file
 
 There is plenty of flexibility to gathering the above information. The sample scripts in the [External builder and launcher sample scripts](#external-builder-and-launcher-sample-scripts) illustrate a simple approach to providing the information.
-As an example of flexibility, consider packaging couchdb index files (see [Add the index to your chaincode folder](https://hyperledger-fabric.readthedocs.io/en/release-1.4/couchdb_tutorial.html#add-the-index-to-your-chaincode-folder)). Sample scripts below describe an approach to packaging the files into code.tar.gz.
+As an example of flexibility, consider packaging couchdb index files (see [Add the index to your chaincode folder](couchdb_tutorial.html#add-the-index-to-your-chaincode-folder)). Sample scripts below describe an approach to packaging the files into code.tar.gz.
```
Comment (sykesm) [inline docs/source/cc_service.md:31]: Nice. Happy to see relative references. :)

**Review by sykesm — DISMISSED** [review]

**Comment by sykesm** [comment]
LGTM; Up to you if you want to change the master/release-2.0.0 ref before merge.

File: docs/source/Fabric-FAQ.rst (hunk 1 above)
Comment (pamandrejko) [reply docs/source/Fabric-FAQ.rst]: Yes. Thanks Matt. I've updated those now. I need to talk to Dave about links like this across the rest of our docs.

Comment (pamandrejko) [reply docs/source/Fabric-FAQ.rst]: Brett thinks he has a way to dynamically detect the branch so we never have to update these when a branch is cut. Will try to add that tomorrow.

**Review by sykesm — APPROVED** [review]

**Pull request merged**